FiveCalls/FiveCalls/IssuesContainerViewController.swift

```diff
@@ -38,15 +39,40 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
         
         issuesVC.didMove(toParentViewController: self)
         issuesViewController = issuesVC
+        
+        let effectView = UIVisualEffectView(frame: headerContainer.bounds)
+        effectView.translatesAutoresizingMaskIntoConstraints = false
+        effectView.effect = UIBlurEffect(style: .light)
+        
+        headerView.translatesAutoresizingMaskIntoConstraints = false
+        effectView.contentView.addSubview(headerView)
+        headerContainer.addSubview(effectView)
+        
+        NSLayoutConstraint.activate([
+            effectView.contentView.topAnchor.constraint(equalTo: headerView.topAnchor),
+            effectView.contentView.bottomAnchor.constraint(equalTo: headerView.bottomAnchor),
+            effectView.contentView.leftAnchor.constraint(equalTo: headerView.leftAnchor),
+            effectView.contentView.rightAnchor.constraint(equalTo: headerView.rightAnchor),
+            
+            headerContainer.topAnchor.constraint(equalTo: effectView.topAnchor),
+            headerContainer.bottomAnchor.constraint(equalTo: effectView.bottomAnchor),
+            headerContainer.leftAnchor.constraint(equalTo: effectView.leftAnchor),
+            headerContainer.rightAnchor.constraint(equalTo: effectView.rightAnchor)
+            ])
```

Comment:
BTW I was wondering why you didn't set up the container/contained relationship directly in the storyboard... Figured you had a reason but didn't investigate.